.devops/template/make-build.yml

```diff
@@ -25,6 +25,7 @@ steps:
     NEXT_PUBLIC_ANALYTICS_LOG_IP: $(TF_PUBLIC_ANALYTICS_LOG_IP)
     NEXT_PUBLIC_ANALYTICS_DEBUG: $(TF_PUBLIC_ANALYTICS_DEBUG)
     NEXT_PUBLIC_SPID_TEST_ENV_ENABLED: $(TF_PUBLIC_SPID_TEST_ENV_ENABLED)
+    NEXT_PUBLIC_SPID_TEST_ENV_UAT_ENABLED: 'true'
```

Comment:
We put `true` hardcoded?

Reply:
This is an env var that will be deleted after the validation step has been completed. I put this hardcoded here to prevent another PR on the iac side and to facilitate the removal.